src/libraries/System.Data.Odbc/src/Common/System/Data/Common/AdapterUtil.Odbc.cs

```diff
@@ -10,6 +10,8 @@
 using System.Runtime.Versioning;
 using System.Text;
 
+#pragma warning disable CA2249 // Consider using 'string.Contains' instead of 'string.IndexOf'
```

Comment:
We're suppressing it because it builds for some configurations that lack the relevant overload?  Would it be better then to suppress the rule with NoWarn for the whole project?

Reply:
Yes for the first question. I'll add the rule to the nowarn tags in the csproj